prov/efa/src/rxr/rxr_init.c

```diff
@@ -429,6 +429,30 @@ int rxr_get_lower_rdm_info(uint32_t version, const char *node,
 	return ret;
 }
 
+static int rxr_shm_space_check(void)
+{
+	struct statvfs stat;
+	char shm_fs[] = "/dev/shm";
+	uint64_t size;
+	int ret = 0;
```

Comment:
I prefer change ```ret```to ```err``` to be more specific.

Reply:
Done.